CHANGELOG

```diff
@@ -1,3 +1,10 @@
+* Fri Jun 14 2019 Michael Morrone <michael.morrone@onyxpoint.com> - 0.1.2
```

Comment:
- No version bump required
- Please combine into one 0.1.1 changelog
- Please put details of interest to end-user first